deploy/cron.js

```diff
@@ -86,10 +86,10 @@ const nextRunTime = new Date(
 	currentTime.getDate() + 1,
 	10 - currentTime.getTimezoneOffset() / 60
 );
-setInterval(() => {
+setTimeout(() => {
```

Comment:
So, this is what it should have been all along. The first block runs once, and then sets the interval so the next block runs every 24 hours.